packages/cli/package.json

```
@@ -0,0 +1,70 @@
+  "bin": {
+    "relative-ci": "bin/index.js"
+  },
+  "scripts": {
+    "build": "npm run clean && tsc && rollup -c && npm run build-type",
+    "build-type": "echo '{\"type\":\"commonjs\"}' > lib/cjs/package.json && echo '{\"type\":\"module\"}' > lib/esm/package.json",
+    "clean": "rimraf ./lib",
+    "lint": "eslint .",
+    "test": "vitest src --passWithNoTests --run"
+  },
+  "devDependencies": {
+    "@types/fs-extra": "11.0.4",
```

Biome (1.9.4)

[error] The key @types/fs-extra was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)